src/deion/worker/core/player/value.js

```diff
@@ -58,37 +66,51 @@ const value = (
 	const slope = 1.531;
 	const intercept = 31.693;
 
-	// 1. Account for stats (and current ratings if not enough stats)
+	// 1. OVERALL RATINGS + PER + CONSISTENCY
+	// Account for stats (and current ratings if not enough stats)
 	const ps = p.stats.filter(playerStats => !playerStats.playoffs);
 	let current = pr.ovr; // No stats at all? Just look at ratings more, then.
 	if (process.env.SPORT === "basketball" && ps.length > 0) {
 		const ps1 = ps[ps.length - 1]; // Most recent stats
-
-		if (ps.length === 1 || ps[0].min >= 2000) {
+		if (ps.length === 1) {
 			// Only one year of stats
-			current = intercept + slope * ps1.per;
-			if (ps1.min < 2000) {
-				current = (current * ps1.min) / 2000 + pr.ovr * (1 - ps1.min / 2000);
+			if (ps1.min > 14 * ps1.gp) {
+				//high mpg
+				current = 0.6 * current + 0.4 * ps1.per;
+			} else {
+				//low mpg
+				current = 0.875 * current + 0.125 * ps1.per;
 			}
 		} else {
-			// Two most recent seasons
-			const ps2 = ps[ps.length - 2];
-			if (ps1.min + ps2.min > 0) {
-				current =
-					intercept +
-					(slope * (ps1.per * ps1.min + ps2.per * ps2.min)) /
-						(ps1.min + ps2.min);
-
-				if (ps1.min + ps2.min < 2000) {
-					current =
-						(current * (ps1.min + ps2.min)) / 2000 +
-						pr.ovr * (1 - (ps1.min + ps2.min) / 2000);
-				}
+			const ps2 = ps[ps.length - 2]; // Two most recent seasons
+			const PER2OVR =
+				intercept +
+				((ps1.per * ps1.min + ps2.per * ps2.min) / (ps1.min + ps2.min)) * slope;
+			if (ps1.min + ps2.min > 1250) {
+				//if they've played significant minutes
+				current = 0.6 * current + 0.4 * PER2OVR;
+			} else {
+				current = 0.875 * current + 0.125 * PER2OVR;
 			}
 		}
-		current = 0.1 * pr.ovr + 0.9 * current; // Include some part of the ratings
 	}
 
+	/*
+	//lets value differently for w/ contract vs w/o
+	if(options.withContract){
```

Comment:
If you're going to add something like this here, call it `p.valueWithContract`. I used to have something like that, which was where the `withContract` option came from. Only add it if you think it makes the trade code easier or clearer.

Reply:
I played around with it, I don't think it has much value for trades, I will remove the giant comment I left.